database/seed.js

```diff
@@ -0,0 +1,45 @@
+const mysql = require('mysql');
+const moment = require('moment');
+const restaurantData = require('./restaurantData.js');
+const restaurants = restaurantData.restaurantData;
+const db=require('./model.js');
+
+const connection = mysql.createConnection({
+    user: 'root',
+    password: '',
+    database: 'yelp'
+  });
+  
+  connection.connect();
+
+module.exports={
+    seedRestaurant: () => {
+        const list = [];
+        console.log(restaurants.length-1);
+        for (var i = 0; i <= 100; i++) {
```

Comment:
Will result in 101 iterations: intended result?

Reply:
fixed